addon/components/layer-tree-headers/base.js

```diff
@@ -0,0 +1,17 @@
+/**
+  @module ember-flexberry-gis
+*/
+
+import Ember from 'ember';
+import DynamicActionsMixin from 'ember-flexberry/mixins/dynamic-actions';
+
+/**
+  Base layer tree header component.
+  Layer components should be inheriting from this.
+
+  @class BaseLayerTreeHeaderComponent
```

Comment:
I think name TreenodeContent better corresponds the aim of component. It is not actually header, and it is not in tree, it is in treenode.